core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -75,6 +75,8 @@ class TaskInfo(
   }
 
   private[spark] def markFinished(state: TaskState, time: Long = System.currentTimeMillis) {
```

Comment:
How about removing the default value for the argument? I only see one other call site for this method that would need to be changed.